src/protocolsupport/protocol/serializer/ItemStackSerializer.java

```diff
@@ -285,6 +287,18 @@ public static NetworkItemStack remapItemToClient(ProtocolVersion version, String
 		return ItemStackRemapper.remapToClient(version, locale, itemstack);
 	}
 
+	public static NBTCompound itemStackToPENBT(ProtocolVersion version, String locale, NetworkItemStack itemstack) {
```

Comment:
The proper place for it is the CommonNBT class

Reply:
@Shevchik Indeed! The function was very similar to the pre-existing serializeItemStackToNBT. I renamed it to serializeItemStackToPENBT to highlight this similarity.

It is possible that the two functions could be united more, but they seem to do slightly different remapping and stuff, so I thought at the point trying to unite what was similar would just make it harder to read.